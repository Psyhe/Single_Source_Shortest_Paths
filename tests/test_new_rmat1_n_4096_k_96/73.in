4096 3130 3171
3131 2098 107
3132 2257 106
3132 2178 253
3133 3368 251
3134 18 179
3134 548 254
3136 514 182
3136 320 52
3136 256 67
3136 768 246
3136 2121 45
3136 1 10
3136 140 96
3136 542 104
3136 3024 183
3136 681 127
3136 2112 50
3136 641 155
3136 128 139
3136 20 179
3136 292 143
3136 4 107
3136 3088 201
3136 1 238
3136 192 63
3136 0 27
3136 1104 241
3136 2064 106
3136 580 72
3136 169 6
3136 1092 93
3136 10 90
3136 2072 201
3136 512 32
3136 1570 153
3136 576 98
3136 2064 21
3136 260 6
3136 2048 171
3136 1080 26
3136 2102 94
3136 776 252
3136 32 187
3136 1155 223
3136 262 226
3136 396 175
3136 2200 200
3136 1040 180
3136 3211 190
3136 153 88
3136 273 225
3136 641 84
3136 2368 73
3136 512 192
3136 516 44
3136 68 240
3136 2053 217
3136 547 231
3136 48 213
3136 324 216
3136 18 111
3136 104 19
3136 2204 181
3136 1698 43
3136 321 63
3136 0 184
3136 784 189
3136 1544 145
3136 128 158
3136 2067 225
3136 552 230
3136 412 39
3136 0 229
3136 1678 166
3136 105 64
3136 640 146
3136 128 215
3136 1100 1
3136 800 223
3136 3212 232
3136 32 217
3137 3 213
3137 2048 80
3137 256 241
3137 4 199
3137 3585 87
3137 72 215
3137 260 240
3137 1654 121
3137 134 77
3137 547 143
3137 80 98
3137 2592 151
3137 8 96
3137 2368 59
3137 332 157
3137 80 123
3137 1800 180
3137 48 57
3137 1412 151
3137 1280 67
3138 2 165
3138 112 173
3138 808 173
3138 1477 70
3138 2072 40
3138 2 105
3138 913 253
3138 8 59
3138 69 234
3138 776 53
3138 968 40
3138 24 154
3138 290 229
3138 198 243
3138 264 142
3138 260 133
3138 10 148
3139 34 206
3139 672 189
3139 10 93
3139 128 148
3139 1546 218
3139 2178 46
3140 8 143
3140 1596 132
3140 2048 43
3140 1106 5
3140 137 176
3140 806 95
3140 147 185
3140 2305 167
3140 19 4
3140 1025 159
3140 2 8
3140 34 166
3140 1074 163
3140 12 70
3140 278 71
3140 92 153
3140 83 124
3140 3 175
3140 2737 4
3140 3074 190
3140 2320 58
3140 130 121
3140 4 78
3140 1600 213
3140 2080 59
3140 1536 139
3141 0 253
3141 74 134
3141 64 11
3141 16 166
3141 608 237
3141 610 241
3141 257 247
3141 0 12
3141 2136 60
3141 257 77
3141 1323 149
3141 160 234
3142 3200 208
3142 161 105
3142 1024 1
3142 2 32
3143 576 129
3144 8 231
3144 786 59
3144 620 66
3144 2318 120
3144 2372 127
3144 807 181
3144 2421 198
3144 392 19
3144 133 177
3144 416 123
3144 132 238
3144 450 197
3144 2080 20
3144 224 55
3144 192 50
3144 149 223
3144 408 167
3144 672 6
3144 145 86
3144 37 214
3144 2177 157
3144 4 205
3144 392 204
3144 128 213
3144 788 239
3145 932 208
3145 896 121
3145 2080 162
3145 128 142
3145 2261 143
3145 2610 104
3145 204 220
3145 128 87
3145 3712 236
3145 584 45
3145 1536 175
3145 28 15
3146 2080 192
3146 514 161
3146 864 175
3146 16 145
3147 3072 79
3147 64 2
3148 146 16
3148 88 221
3148 0 220
3148 1024 192
3148 320 128
3148 4 187
3148 1 161
3148 50 41
3149 16 139
3149 15 20
3150 32 94
3150 528 246
3152 16 137
3152 3584 192
3152 522 219
3152 0 129
3152 2048 78
3152 1440 145
3152 53 99
3152 24 150
3152 449 59
3152 225 207
3152 2048 38
3152 66 242
3152 64 224
3152 5 189
3152 192 15
3152 2176 61
3152 2048 185
3152 3584 11
3152 258 51
3152 0 17
3152 779 174
3152 1542 188
3152 32 29
3152 770 43
3152 3648 98
3152 64 193
3152 1541 61
3152 140 115
3152 30 38
3152 256 148
3152 644 214
3152 16 218
3152 97 130
3152 172 91
3152 2 44
3153 150 92
3153 5 147
3153 64 142
3153 266 20
3153 2 167
3153 2688 65
3153 1032 2
3154 96 230
3154 1032 69
3154 1026 45
3154 321 130
3154 1280 109
3154 1152 43
3154 1105 31
3154 512 192
3154 3073 84
3155 3232 67
3155 1033 14
3155 800 116
3156 0 10
3156 68 52
3156 0 81
3157 8 136
3157 2088 81
3157 1059 213
3157 709 30
3158 33 50
3160 2114 104
3160 912 21
3160 0 22
3160 4 82
3160 16 68
3160 400 144
3160 1696 68
3160 257 178
3161 1280 225
3161 2613 11
3161 547 54
3161 2224 142
3161 544 247
3162 20 172
3163 264 127
3164 352 222
3166 3 35
3167 514 87
3167 1028 158
3168 768 226
3168 516 94
3168 35 139
3168 3864 215
3168 0 60
3168 3648 35
3168 1072 30
3168 1424 169
3168 660 184
3168 85 130
3168 0 183
3168 516 19
3168 258 201
3168 149 237
3168 1560 218
3168 77 115
3168 2560 235
3168 2304 101
3168 1056 153
3168 2052 69
3168 18 101
3168 1105 245
3169 1792 224
3169 514 179
3169 48 142
3169 3586 207
3169 1296 86
3169 16 135
3169 1024 66
3169 2308 111
3170 97 139
3170 2064 73
3170 0 12
3170 132 196
3170 664 60
3170 120 178
3170 4 77
3170 784 43
3171 12 50
3171 1028 173
3171 3124 13
