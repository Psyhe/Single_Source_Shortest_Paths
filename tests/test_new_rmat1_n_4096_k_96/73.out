inf
129
157
300
182
inf
6
62
52
49
9
12
2
137
7
23
147
3
21
45
95
inf
17
4
42
20
10
86
51
inf
22
69
174
130
235
inf
39
94
21
67
12
28
